extension/src/test/e2e/extension.test.ts

```diff
@@ -11,184 +10,190 @@ import {
+  it('should show the DVC Icon', async function () {
+    const dvcIcon = await getDVCActivityBarIcon()
+    expect(await dvcIcon.getTitle()).toBe('DVC')
   })
+})
 
-  describe('Experiments Table Webview', () => {
-    const webview = new ExperimentsWebview('experiments')
-
-    it('should load as an editor', async () => {
-      const workbench = await browser.getWorkbench()
+describe('Experiments Table Webview', function () {
+  const webview = new ExperimentsWebview('experiments')
 
-      await workbench.executeCommand('DVC: Show Experiments')
+  const epochs = 15
+  const headerRows = 3
+  const workspaceRow = 1
+  const commitRows = 3
+  const initialRows = headerRows + workspaceRow + commitRows
 
-      await webview.focus()
+  it('should load as an editor', async function () {
+    const workbench = await browser.getWorkbench()
 
-      await browser.waitUntil(async () => {
-        const table = await webview.table$
-        return table.isDisplayed()
-      })
+    await workbench.executeCommand('DVC: Show Experiments')
 
-      expect(await webview.table$$).toHaveLength(1)
+    await webview.focus()
 
-      await webview.unfocus()
+    await browser.waitUntil(async () => {
+      const table = await webview.table$
+      return table.isDisplayed()
     })
 
-    it('should update with a new row for each checkpoint when an experiment is running', async () => {
-      const epochs = 15
-      const headerRows = 3
-      const workspaceRow = 1
-      const commitRows = 3
-      const initialRows = headerRows + workspaceRow + commitRows
-
-      await webview.focus()
-      await browser.waitUntil(
-        async () => {
-          await webview.expandAllRows()
-          const currentRows = await webview.row$$
-          return currentRows.length === initialRows
-        },
-        { interval: 5000, timeout: 30000 }
-      )
-
-      await webview.unfocus()
-      await runModifiedExperiment()
-      await webview.focus()
-
-      await browser.waitUntil(
-        async () => {
-          await webview.expandAllRows()
-          const currentRows = await webview.row$$
-          return currentRows.length >= initialRows + epochs
-        },
-        { interval: 5000, timeout: 180000 }
-      )
-
-      await webview.unfocus()
-      await waitForDvcToFinish()
-      await webview.focus()
-
-      const finalRows = await webview.row$$
-
-      expect(finalRows.length).toStrictEqual(initialRows + epochs)
-      await webview.unfocus()
-      await waitForDvcToFinish()
-      const workbench = await browser.getWorkbench()
-      await workbench.executeCommand('Terminal: Kill All Terminals')
-    }).timeout(180000)
+    expect(await webview.table$$).toHaveLength(1)
+
+    return webview.unfocus()
   })
 
-  describe('Plots Webview', () => {
-    before(async () => {
-      await closeAllEditors()
-    })
+  it('should show the correct number of records before running an experiment', async function () {
```

Comment:
[F] This test has to be split in half or the framework cannot find the command palette.